au/code/au/utility/test/string_constant_test.cc

```diff
@@ -21,13 +21,14 @@
 namespace au {
 
 using ::testing::Eq;
+using ::testing::IsEmpty;
 using ::testing::StrEq;
 
 namespace detail {
 
 TEST(StringConstant, CanCreateFromStringLiteral) {
     constexpr StringConstant<5> x{"hello"};
-    EXPECT_STREQ(x.c_str(), "hello");
```

Comment:
I didn't realize we could get rid of these `c_str()` calls.  Neat!  I wonder if we needed them with `EXPECT_STREQ`, and if not, then I wonder why I added them in the first place.

Reply:
I'm not sure I tried it.  I do know the matcher certainly provided better ability to match different types, so I just tried it and it worked!